blog/journey-to-d14n.md

```diff
@@ -0,0 +1,111 @@
+---
+slug: journey-to-decentralization
+hide_table_of_contents: false
+title: "XMTP: The journey to decentralization"
+date: 2023-05-24
+authors: montez
+description: "XMTP Labs is taking a phased approach toward architectural and logical decentralization of the network. The mission is to make sure the network is permissionless: owned by infrastructure providers and users that choose to participate, which can be anyone. Taking it further, these permissionless infra providers and users will become symbiotic, since one can’t exist without the other."
+tags:
+- Decentralization
+---
+
+# XMTP: The journey to decentralization
+
+XMTP Labs is taking a phased approach toward architectural and logical decentralization of the XMTP network. The mission is to make sure the network is permissionless: owned by infrastructure providers and users that choose to participate, which can be anyone. Taking it further, these permissionless infra providers and users will become symbiotic, since one can’t exist without the other. 
```

Comment:
I changed "We are" to "XMTP Labs is" - just to be clear and transparent about who is taking the approach. We may even want to mention the current state of the network as a way to introduce where we are going?

This is how we describe the current state of the network in the FAQ: Currently, XMTP Labs (the company) operates all of the network nodes in the two available XMTP network environments: `dev` and `production`.